solidity/contracts/BancorNetwork.sol

```diff
@@ -260,6 +273,24 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, F
         return toAmount;
     }
 
+    function change(
+        IBancorConverter _converter,
+        IERC20Token _fromToken,
+        IERC20Token _toToken,
+        uint256 _amount,
+        uint256 _minReturn
+    ) private returns (uint256) {
+        if (etherTokens[_fromToken] || etherTokens[_toToken]) {

```

Comment:
@yudilevi : Yes, for saving gas. Otherwise, `isETHConverter(_converter)`, which yields an external function-call, is always invoked (i.e., not only when it needs to be).

Reply:
ok